operators/math/math.cc

```diff
@@ -6,12 +6,14 @@
 #include "segment_extraction.hpp"
 #include "segment_sum.hpp"
 
-
-FxLoadCustomOpFactory LoadCustomOpClasses_Math = 
-    LoadCustomOpClasses<CustomOpClassBegin, 
-                        CustomOpNegPos,
+const std::vector<const OrtCustomOp*>& MathLoader() {
+  static OrtOpLoader op_loader(LiteCustomOp("NegPos", neg_pos),
```

Comment:
can all members which are stored in OrtOpLoader be released without any ABI from ORT?